provableAPI_0.6.sol

```diff
@@ -0,0 +1,1326 @@
+// <provableAPI>
+/*
+Copyright (c) 2015-2016 Oraclize SRL
+Copyright (c) 2016-2019 Oraclize LTD
+Copyright (c) 2019-2020 Provable Things Limited
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+*/
+pragma solidity >= 0.5.0 < 0.7.0; // Incompatible compiler version - please select a compiler within the stated pragma range, or use a different version of the provableAPI!
+
+// Dummy contract only used to emit to end-user they are using wrong solc
+abstract contract solcChecker {
+/* INCOMPATIBLE SOLC: import the following instead: "github.com/oraclize/ethereum-api/oraclizeAPI_0.4.sol" */ function f(bytes calldata x) virtual external;
+}
+
+interface ProvableI {
```

Comment:
The keyword `interface` seems much more correct here, did you add it because it wasn’t compiling anymore?

Reply:
Solc 0.6 adds new requirements around explicitly declare interfaces as such. This change is necessary and expected since this is an interface after all (the interface keyword didn't exist when the provableAPI was first written)